beacon_node/network/src/service.rs

```diff
@@ -270,7 +274,15 @@ impl<T: BeaconChainTypes> NetworkService<T> {
         };
 
         // launch libp2p service
-        let (mut libp2p, network_globals) = Network::new(executor.clone(), service_context).await?;
+        let (mut libp2p, network_globals) = Network::new(
+            executor.clone(),
+            service_context,
+            beacon_chain
+                .data_availability_checker
+                .custody_context()
+                .sampling_count(&beacon_chain.spec),
```

Comment:
should this be the advertised custody group count ?

Reply:
Addressed in https://github.com/sigp/lighthouse/pull/7578/commits/cd7059c34b33c5435820a2cf3869287a7be6ad0e

We should pass CGC here, as the sampling count is derived in the function later.